docs/main-courses/pastas/pasta/pastasallad.md

```diff
@@ -0,0 +1,18 @@
+Pastasallad
```

Comment:
Here you created a file in the directory docs/main-courses/pastas/pasta/ . I think it's fine to have it a level above, that is docs/main-courses/pastas/.
This way the link to the specific recipe would be docs/main-courses/pastas/pastasallad.md

This way it would reflect what you added in the mkdocs.yml file!